packages/link/src/components/Link.js

```diff
@@ -35,7 +35,7 @@ const Link = props => {
     displayBlock
   };
 
-  const componentType = linkProps.href ? "a" : "div";
+  const componentType = linkProps.href ? "a" : "span";
```

Comment:
this is a breaking change, please change the commit message so semantic version can bump the major version of the package when published

Reply:
ok, @vNakamura explained on Slack that the broken behaviuor was introduced earlier and this is actually the fix